src/control/common/proto/types_test.go

```diff
@@ -120,6 +121,32 @@ func TestProto_ConvertScmModules(t *testing.T) {
 	}
 }
 
+func TestProto_ConvertScmMountPoint(t *testing.T) {
+	a := storage.MockScmMountPoint()
+	resA, _ := json.Marshal(a)
```

Comment:
nit: Probably better to check the err and `t.Fatal(err)` if non-nil, just to avoid confusing test results if the marshal starts failing later due to some code change.

Reply:
will fix in a subsequent PR, thanks

Reply:
fix in https://github.com/daos-stack/daos/pull/6366